### DISCUSSION TITLE
No screen transition when signing in with Google in Flutter (Web)

### DISCUSSION THREAD

**Discussion by tm20314** [supabase/supabase#27380] [Questions] [answered]
# Introduction.
Nice to meet you. I am a newcomer to Supanase.
I'm Japanese and I use machine translation, so my English is not very good.

As for the main topic, as the title says
When using Flutter (web) and Supabase (Google sign-in)

I can log in, but the subsequent screen transitions do not work properly.

# Currently occurring problems

| Native | Web |
|--------|--------|
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-06-19 at 02 45 20](https://github.com/supabase/supabase/assets/39609331/a44129e6-27bb-4cce-9394-2991ca16cdbc) | ![CleanShot 2024-06-19 at 02 46 31](https://github.com/supabase/supabase/assets/39609331/96be6404-7781-4087-8e56-62b754023724) |

As shown in the image, native (iOS/Android) screen transitions occur after the login process, but
However, the screen transition is not performed on the web, and the user is returned to the login screen.

# What we did.

The following information was helpful in addressing this issue.
[How to make a rough application]
https://supabase.com/blog/flutter-authentication
[How to make a rough application (video version)
https://www.youtube.com/watch?v=utMg6fVmX0U
[Documentation]
https://supabase.com/docs/guides/auth/social-login/auth-google?queryGroups=platform&platform=flutter

# About the source code
I will send you the source code as a courtesy.

The UI and processing of the application endpoints and the connection/login screen with Supabase are the following files.

https://github.com/gadgelogger/count_app/blob/main/lib/main.dart

The screen (destination) to which the user transitions after the login process is the following file.

https://github.com/gadgelogger/count_app/blob/main/lib/screens/second_screen.dart

# At the end.
Supabase integration and login process
etc. are completed and working.
I am asking this question because I really don't know the cause of the problem.
I am sorry for the length of this question, but I would appreciate it if you could enjoy it when you are free. Thank you very much.

**Comment by tinkie101** [comment, marked as answer]
Hi, I'm not sure if it's the same problem that I had.

But there was a bug with Flutter Web auth. It got fixed in supabase_flutter version 2.5.9. Try upgrading and see if that solves your problem

Hope this helps :)
